webapp/src/components/modals/channel_subscriptions/channel_subscriptions.tsx

```diff
@@ -18,7 +16,7 @@ export default class ChannelSubscriptionsModal extends PureComponent<Props> {
     state = {
         showModal: false,
         allProjectMetadata: null,
-    }
+    };
```

Comment:
This component has state, but there is no `State` generic being provided to the `PureComponent`. I wonder if this is something the typescript linter can catch. Either way, we should probably add the `State` type to the component